media/views/snykCode/suggestion/suggestionLS.scss

```diff
@@ -1,5 +1,9 @@
 @import '../../common/webview';
 
+.data-flow-number{
+  background-color: var(--vscode-editor-background);
+  background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.075), rgba(255, 255, 255, 0.075));
```

Comment:
can't this be pushed down to LS?

Reply:
Also, does the hardcoded gradient work on light and dark? I suppose we should use variables here :)

Reply:
I couldn't find a variable that would work with different themes, without adding anything else to VSCode, I could add the change but on light mode, the background of the bubbles would be. I could use variables but I think they would probabbly still need to be defined in VSCode.

Reply:
If this is still fine, I can add it to Language Server, but if we want to match the figma more, the bubbles should have the general background of the panel.
<img width="581" alt="Screenshot 2024-12-13 at 11 51 53" src="https://github.com/user-attachments/assets/d1580df1-8f4c-4039-81de-8b7520d226e2" />

Reply:
I removed the `background-color` attribute and left it in LS, if we specifically don't want to agglomerate CSS files, we could replace the variable depending on the theme, but I'm not sure if there would be much difference.